core/db/migrate/20210902123011_create_spree_wished_products.rb

```diff
@@ -0,0 +1,16 @@
+class CreateSpreeWishedProducts < ActiveRecord::Migration[5.2]
+  def change
+    create_table :spree_wished_products, if_not_exists: true do |t|
+      t.references :variant
+      t.belongs_to :wishlist
+
+      t.column :quantity, :integer, default: 1, null: false
+      t.column :remark, :text
+
+      t.timestamps
+    end
+
+    add_index :spree_wished_products, :variant_id unless index_exists?(:spree_wished_products, :variant_id)
+    add_index :spree_wished_products, :wishlist_id unless index_exists?(:spree_wished_products, :wishlist_id)
```

Comment:
@MatthewKennedy we also need an additional migration adding unique index on variant_id + wishlist_id

Reply:
Will do that.

Reply:
Current state of indexes:
```ruby
  create_table "spree_wished_items", force: :cascade do |t|
    t.bigint "variant_id"
    t.bigint "wishlist_id"
    t.integer "quantity", default: 1, null: false
    t.datetime "created_at", null: false
    t.datetime "updated_at", null: false
    t.index ["variant_id", "wishlist_id"], name: "index_spree_wished_items_on_variant_id_and_wishlist_id", unique: true
    t.index ["variant_id"], name: "index_spree_wished_items_on_variant_id"
    t.index ["wishlist_id"], name: "index_spree_wished_items_on_wishlist_id"
  end

  create_table "spree_wishlists", force: :cascade do |t|
    t.bigint "user_id"
    t.bigint "store_id"
    t.string "name"
    t.string "token", null: false
    t.boolean "is_private", default: true, null: false
    t.boolean "is_default", default: false, null: false
    t.datetime "created_at", null: false
    t.datetime "updated_at", null: false
    t.index ["store_id"], name: "index_spree_wishlists_on_store_id"
    t.index ["token"], name: "index_spree_wishlists_on_token", unique: true
    t.index ["user_id", "is_default"], name: "index_spree_wishlists_on_user_id_and_is_default"
    t.index ["user_id"], name: "index_spree_wishlists_on_user_id"
  end
```